miniapps/zora-nft-minter/src/manifest.ts

```diff
@@ -0,0 +1,27 @@
+import { ModManifest } from "@mod-protocol/core";
+import view from "./view";
+
+const manifest: ModManifest = {
+  slug: "zora-nft-minter",
+  name: "Preview and mint Zora NFTs",
+  custodyAddress: "stephancill.eth",
+  version: "0.0.1",
+  logo: "https://i.imgur.com/fuSkgoJ.png",
+  custodyGithubUsername: "stephancill",
+  contentEntrypoints: [
+    {
+      if: {
+        value: "{{embed.url}}",
+        match: {
+          contains: "zora.co/collect",
```

Comment:
could false positive match if this url is in the query parameter of another, but likely fine for now